manager/client/acquire/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"os"
+	"time"
+	"log"
+	"context"
+	"google.golang.org/grpc"
+	"strconv"
+	rpc_license "github.com/enfabrica/enkit/manager/rpc"
+)
+
+func main() {
+	timeout := 1800 * time.Second
+	interval := 60 * time.Second
```

Comment:
For argv parsing, the rest of the codebase here is using cobra with a bit of magic plumbing. For this command specificially, I recommend you do the same as the astore main, [here](https://github.com/enfabrica/enkit/blob/master/astore/server/main.go#L252). If you want to, we can look at it together, but if you follow the advice above, you can use the same pattern with `DefaultFlags` to register the `retry` library flags as well, so the attempts etc are configured from the command line.

Reply:
(I actually recommend you do those changes last, my first recommendation would be to focus on the interface below, and mechanics above)